index.js

```diff
@@ -67,6 +81,31 @@ module.exports = function (src, dest, opts) {
 				resolve(true);
 			});
 
+			var progressEnabled = progress.emitter.listeners('progress').length > 0;
+			if (progressEnabled) {
```

Comment:
Size would be `0` only if it is empty file, I think there would be no data events in this case, and we can just report progress as 1.
Need to confirm absence of `data` events though